argo-data-submission-wf/main.nf

```diff
@@ -114,11 +121,36 @@ upload_params = [
   *:(params.upload ?: [:])
 ]
 
+process hello_cram {
```

Comment:
what's the usage of this process?

Reply:
Oops. Placeholder functions for testing. I'll remove